src/sections/SideBar/components/Footer/SideBarFooter.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
Here too, please move the footer component outside to the root level so that we can use it in other components just in case